packages/emptystate/package.json

```diff
@@ -10,12 +10,13 @@
     "access": "public"
   },
   "scripts": {
-    "build": "run-s build:svg build:js build:cjs build:css",
+    "build": "run-s build:svg build:js build:cjs build:css copy:svg",
     "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
     "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
     "build:svg": "svgr --filename-case kebab --ext \"dist.js\" -d src/react/illustrations src/svg",
     "build:watch": "npm run build:js -- --watch",
+    "copy:svg": "node copySvgs.js",
```

Comment:
nitpick: i think we've been using dashed filename. i suggest you change `copySvgs.js` to `copy-svgs.js`

Reply:
Will do!